docs/WEIGHT_DRIFT_AUDIT.md

```
@@ -0,0 +1,363 @@
+POST /v1/admin/weight-drift-audits/{id}/approve
+Content-Type: application/json
+
+{
+  "approvalNotes": "Reviewed markets; drift acceptable for now"
+}
+```
+
+**Returns:** Updated audit with status "approved", approvedBy set, approvalNotes saved
+
+**Audit Log Entry:**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)